ios/ios_swift_4_transition_checklist

```diff
@@ -0,0 +1,20 @@
+# Swift 4 Transition Checklist
+
+Provided below is a short checklist of items to complete so your project is ready to make the jump to Swift 4. 
+
+We ask all projects to add the following checklist to the Readme file and complete the necessary items.
+
+Transitioning Swift Wisdom to Swift 4 will be handled collectively by the iOS team. As a result, there is no need to account for Swift Wisdom related work when completing the checklist.  
+
+```
+# Swift 4 Transition Checklist
+
+Is the project going to remain active beyond launch of Swift 4 (Sept/Oct 2017)  - [ ] Yes  - [ ] No
```

Comment:
We also want projects that will be going into maintenance contracts included in this right? Is it worth specifying explicitly you think?